src/DonationForms/resources/styles/design-settings/image.scss

```diff
@@ -216,3 +218,28 @@
     }
 }
 
+/* Two-Panel */
```

Comment:
I think we would want to move any two panel specific styles outside of the base stylesheet and into its own styles.

Reply:
Yes, I think we should be looking at it as the base image stylesheet. 
Resolved: c18b103411730e4bfcc078af9e1afdad87df1830